libs/application/template-api-modules/src/lib/modules/shared/shared.module.ts

```diff
@@ -20,7 +21,7 @@ export class SharedTemplateAPIModule {
       module: SharedTemplateAPIModule,
       imports: [
         ConfigModule.forRoot({
-          load: [configuration],
+          load: [configuration, sharedModuleConfig],
```

Comment:
Remove? Should be enough to load this in the app.module. 
We should also get of the dynamic registers of the SharedTemplateApiModule

Reply:
Yup, thanks.